Dockerfile

```diff
@@ -1,54 +1,32 @@
-FROM mono:6.0
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 
 #MAINTAINER Cayde Dixon <me@cazzar.net>
 ENV PUID=1000 \
-    PGID=100  \
-    TargetFrameworkDirectory=/usr/lib/mono/
```

Comment:
Why this directory?

Reply:
As I said, rewrote.
What should that directory point to now? I will fix it